src/map/clif.cpp

```diff
@@ -1987,7 +1987,7 @@ void clif_walkok( map_session_data& sd ){
 
 /// Notifies clients in an area, that an other visible object is walking.
 /// Note: unit must not be self
-void clif_move(struct unit_data& ud)
+void clif_move( struct unit_data& ud )
```

Comment:
Why do you now also start with this weird spaces inside parenthesis style from Lemon?

Reply:
Just for better readability and trying to unify styles, we can always change it later

Reply:
But if you unify it, why not unify it in a good format? Besides, we said we don't want to change styles unless we change the code line so that blame shows the person who actually changed it.

Reply:
What's wrong with it? And I've just changed it for the lines I actually modified